rules/anchoring-sales-strategy/rule.md

```diff
@@ -0,0 +1,49 @@
+---
+type: rule
+title: Do you use anchoring effectively in your sales strategy?
+uri: anchoring-sales-strategy
+authors:
+  - title: Tanya Leahy
+    url: https://ssw.com.au/people/tanya-leahy
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Michael Smedley
+    url: https://ssw.com.au/people/michael-smedley
+guid: 63bb4c1a-733f-46ee-b3ac-e364eb4ced50
+created: 2024-05-03T18:29:33.0000000Z
+related: []
+redirects: []
+  
+
+---
+
+Anchoring is a powerful psychological tactic in sales that involves setting the stage for negotiation and price discussions by presenting an initial offer or a piece of information that sets the reference point for the rest of the conversation. This tactic can significantly influence the decision-making process and help drive the negotiation in favor of the salesperson. Clients tend to anchor on inital estimates or figures when considering the duration and cost of projects, including software development.
+
+<!--endintro-->
+
+`youtube: https://www.youtube.com/embed/MBlNEsyTKAU`
+**VIDEO: Anchoring when estimating software projects**  
```

Comment:
Hi @Tanya-Leahy 

- [ ] 1. Please change from "VIDEO:" to "Video:"
- [ ] 2. Include the length at the end as per https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-add-the-titles--length-on-video-captions

Reply:
Done